migrations/2019-11-04-030204_users/up.sql

```diff
@@ -1,11 +1,12 @@
 CREATE TABLE users (
-  user_id SERIAL PRIMARY KEY,
-  user_uuid UUID NOT NULL,
-  hash BYTEA NOT NULL,
-  salt VARCHAR(255) NOT NULL,
-  email VARCHAR(120) NOT NULL UNIQUE,
-  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
-  name VARCHAR NOT NULL
+    user_id SERIAL PRIMARY KEY,
+    user_uuid UUID NOT NULL,
+    hash BYTEA NOT NULL,
+    salt VARCHAR(255) NOT NULL,
+    email VARCHAR(120) NOT NULL UNIQUE,
+    role VARCHAR(32) NOT NULL DEFAULT 'user',
```

Comment:
Mainly added the `role` field here, needed for allowing some routes / action for some roles only.